steps/id-and-residency/id.tsx

```diff
@@ -1,20 +1,22 @@
-import { FieldsetLegend } from "lbh-frontend-react/components";
+import { CurrentTenantNames } from "components/CurrentTenantNames";
+import { ImageInput } from "components/ImageInput";
+import {
+  PostVisitActionInputDetails,
+  PostVisitActionInputDetailsProps,
+} from "components/PostVisitActionInputDetails";
+import { RadioButtons } from "components/RadioButtons";
+import keyFromSlug from "helpers/keyFromSlug";
+import { FieldsetLegend } from "lbh-frontend-react";
 import React from "react";
 import {
   ComponentDatabaseMap,
   ComponentWrapper,
   DynamicComponent,
+  StaticComponent,
 } from "remultiform/component-wrapper";
-import { ImageInput } from "../../components/ImageInput";
 import { makeSubmit } from "../../components/makeSubmit";
-import {
-  PostVisitActionInputDetails,
-  PostVisitActionInputDetailsProps,
-} from "../../components/PostVisitActionInputDetails";
-import { RadioButtons } from "../../components/RadioButtons";
 import { ReviewNotes } from "../../components/ReviewNotes";
 import { getRadioLabelFromValue } from "../../helpers/getRadioLabelFromValue";
-import keyFromSlug from "../../helpers/keyFromSlug";
```

Comment:
This needs some looking at. I would have expected it to be the same as the other steps.